basketball_reference_web_scraper/data.py

```diff
@@ -116,6 +116,47 @@ class Position(Enum):
     'VAN': Team.VANCOUVER_GRIZZLIES,
 }
 
+TEAM_TO_TEAM_ABBREVIATION = {
```

Comment:
Is it possible to invert `TEAM_ABBREVIATIONS_TO_TEAM`?

Reply:
kinda problematic as simple dictionary comprehension will make CHARLOTTE_HORNETS return the deprecated abbreviation "CHH".
I should probably have both and refer to CHH when the requested game occurred before deprecation.

Reply:
s

> kinda problematic as simple dictionary comprehension will make CHARLOTTE_HORNETS return the deprecated abbreviation "CHH".
> I should probably have both and refer to CHH when the requested game occurred before deprecation.

Seems irrelevant because the CHH abbreviation existed only from 2003 and before, and there aren't any play-by-play stats for these years.

Reply:
Made inversion with an explicit override for `Team.CHARLOTTE_HORNETS`